# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR ORGANIZATION
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"POT-Creation-Date: 2007-06-05 23:22+Paris, Madrid\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=CHARSET\n"
"Content-Transfer-Encoding: ENCODING\n"
"Generated-By: pygettext.py 1.5\n"


#: gpicsync-GUI.py:127
msgid "Local time correction"
msgstr ""

#: gpicsync-GUI.py:129
msgid "Language"
msgstr ""

#: gpicsync-GUI.py:131
msgid "Configuration file"
msgstr ""

#: gpicsync-GUI.py:133
msgid "&Options"
msgstr ""

#: gpicsync-GUI.py:135
msgid "About..."
msgstr ""

#: gpicsync-GUI.py:137
msgid "&Tools"
msgstr ""

#: gpicsync-GUI.py:138
msgid "EXIF reader"
msgstr ""

#: gpicsync-GUI.py:139
msgid "Geo-Rename pictures"
msgstr ""

#: gpicsync-GUI.py:140 gpicsync-GUI.py:804
msgid "GPX Inspector"
msgstr ""

#: gpicsync-GUI.py:141
msgid "KMZ Generator"
msgstr ""

#: gpicsync-GUI.py:142
msgid "&Help"
msgstr ""

#: gpicsync-GUI.py:151
msgid "Pictures folder"
msgstr ""

#: gpicsync-GUI.py:152
msgid "GPS file"
msgstr ""

#: gpicsync-GUI.py:153
msgid " Synchronise ! "
msgstr ""

#: gpicsync-GUI.py:154 gpicsync-GUI.py:613
msgid "Quit"
msgstr ""

#: gpicsync-GUI.py:155
msgid "Stop"
msgstr ""

#: gpicsync-GUI.py:156
msgid "Clear"
msgstr ""

#: gpicsync-GUI.py:157
msgid "View in Google Earth"
msgstr ""

#: gpicsync-GUI.py:159
msgid "UTC Offset="
msgstr ""

#: gpicsync-GUI.py:160
msgid "Geocode picture only if time difference to nearest track point is below (seconds)="
msgstr ""

#: gpicsync-GUI.py:161
msgid "Create a log file in picture folder"
msgstr ""

#: gpicsync-GUI.py:163
msgid "Dates must match"
msgstr ""

#: gpicsync-GUI.py:165
msgid "Create a Google Earth file"
msgstr ""

#: gpicsync-GUI.py:167
msgid "with TimeStamp"
msgstr ""

#: gpicsync-GUI.py:169
msgid "Google Maps export, folder URL="
msgstr ""

#: gpicsync-GUI.py:173
msgid "backup pictures"
msgstr ""

#: gpicsync-GUI.py:175
msgid "interpolation"
msgstr ""

#: gpicsync-GUI.py:177
msgid "add geonames and geotagged"
msgstr ""

#: gpicsync-GUI.py:266
msgid "Choose a language"
msgstr ""

#: gpicsync-GUI.py:266
msgid "languages choice"
msgstr ""

#: gpicsync-GUI.py:336
msgid "More informations and help:"
msgstr ""

#: gpicsync-GUI.py:353
msgid "To visualize the results in Google Earth you must either:"
msgstr ""

#: gpicsync-GUI.py:354 gpicsync-GUI.py:797
msgid "- finish a synchronisation"
msgstr ""

#: gpicsync-GUI.py:366
msgid "Couldn't find or launch Google Earth"
msgstr ""

#: gpicsync-GUI.py:402
msgid "For information, GPX file created with GPSBabel in your picture folder."
msgstr ""

#: gpicsync-GUI.py:404
msgid "Possible problem with the creation of the gpx file"
msgstr ""

#: gpicsync-GUI.py:407
msgid "Couldn't create the necessary GPX file."
msgstr ""

#: gpicsync-GUI.py:429
msgid "You must first select a pictures folder and a GPX file."
msgstr ""

#: gpicsync-GUI.py:443
msgid "Beginning synchronization with "
msgstr ""

#: gpicsync-GUI.py:444
msgid "UTC Offset ="
msgstr ""

#: gpicsync-GUI.py:445
msgid " hours and maximum time difference = "
msgstr ""

#: gpicsync-GUI.py:445
msgid " seconds"
msgstr ""

#: gpicsync-GUI.py:455
msgid "Creating an 'originals-backup' folder."
msgstr ""

#: gpicsync-GUI.py:462
msgid "Starting to generate a Google Earth file (doc.kml) in the picture folder ..."
msgstr ""

#: gpicsync-GUI.py:466
msgid "Starting to generate a Google Map file (doc-web.kml) in the picture folder"
msgstr ""

#: gpicsync-GUI.py:477
msgid "Geocoded with UTC Offset= "
msgstr ""

#: gpicsync-GUI.py:478
msgid " and  Maximum time difference = "
msgstr ""

#: gpicsync-GUI.py:480
msgid "Pictures Folder: "
msgstr ""

#: gpicsync-GUI.py:481
msgid "GPX file: "
msgstr ""

#: gpicsync-GUI.py:497
msgid "(Found "
msgstr ""

#: gpicsync-GUI.py:509
msgid "Processed image "
msgstr ""

#: gpicsync-GUI.py:533
msgid "Couldn't retrieve geonames data..."
msgstr ""

#: gpicsync-GUI.py:556
msgid " (writing geonames and geotagged to keywords tag in picture EXIF)"
msgstr ""

#: gpicsync-GUI.py:564
msgid "FINISHED GEOCODING PROCESS"
msgstr ""

#: gpicsync-GUI.py:566
msgid "PROCESSING STOPPED BY THE USER"
msgstr ""

#: gpicsync-GUI.py:570
msgid "Adding the GPS track log to the Google Earth kml file"
msgstr ""

#: gpicsync-GUI.py:573
msgid "Click on the 'View in Google Earth' button to visualize the result"
msgstr ""

#: gpicsync-GUI.py:574
msgid "( A Google Earth doc.kml file has been created in your picture folder.)"
msgstr ""

#: gpicsync-GUI.py:580
msgid "( A Google Maps doc-web.kml file has been created with the given url )"
msgstr ""

#: gpicsync-GUI.py:600
msgid "Local time corrections"
msgstr ""

#: gpicsync-GUI.py:603
msgid "Use this option ONLY if your camera local time is wrong."
msgstr ""

#: gpicsync-GUI.py:604
msgid "Indicate here the local time now displayed by your camera and GPS (hh:mm:ss)"
msgstr ""

#: gpicsync-GUI.py:606
msgid "Local time displayed now by the camera"
msgstr ""

#: gpicsync-GUI.py:609
msgid "Local time displayed now by the GPS"
msgstr ""

#: gpicsync-GUI.py:612
msgid "Apply correction"
msgstr ""

#: gpicsync-GUI.py:633
msgid "EXIF Reader"
msgstr ""

#: gpicsync-GUI.py:636
msgid "Read the EXIF metadata of the selected picture."
msgstr ""

#: gpicsync-GUI.py:638 gpicsync-GUI.py:639 gpicsync-GUI.py:670
msgid "All EXIF metadata"
msgstr ""

#: gpicsync-GUI.py:640 gpicsync-GUI.py:674
msgid "Date/Time/Lat./Long."
msgstr ""

#: gpicsync-GUI.py:646
msgid "Select a picture"
msgstr ""

#: gpicsync-GUI.py:668
msgid "Selected metada "
msgstr ""

#: gpicsync-GUI.py:680
msgid "lat./long."
msgstr ""

#: gpicsync-GUI.py:682
msgid "None"
msgstr ""

#: gpicsync-GUI.py:688
msgid "Renaming tool"
msgstr ""

#: gpicsync-GUI.py:691
msgid "This tool renames your pictures with the "
msgstr ""

#: gpicsync-GUI.py:691
msgid "original time/date and lat./long.(if present)"
msgstr ""

#: gpicsync-GUI.py:693
msgid "Rename pictures in a folder"
msgstr ""

#: gpicsync-GUI.py:694
msgid "Rename a single picture"
msgstr ""

#: gpicsync-GUI.py:712 gpicsync-GUI.py:733
msgid "Beginning renaming..."
msgstr ""

#: gpicsync-GUI.py:720 gpicsync-GUI.py:747
msgid "Didn't find Original Time/Date for "
msgstr ""

#: gpicsync-GUI.py:723 gpicsync-GUI.py:754
msgid "Renamed "
msgstr ""

#: gpicsync-GUI.py:737
msgid "Interrupted by the user"
msgstr ""

#: gpicsync-GUI.py:755
msgid "Finished"
msgstr ""

#: gpicsync-GUI.py:762
msgid "Create a kmz file archive"
msgstr ""

#: gpicsync-GUI.py:764
msgid "Create KMZ file !"
msgstr ""

#: gpicsync-GUI.py:772
msgid "Sorry this tool is not yet available for the Linux version"
msgstr ""

#: gpicsync-GUI.py:781
msgid "Creating a KMZ file in the pictures folder..."
msgstr ""

#: gpicsync-GUI.py:784
msgid "Adding doc.kml"
msgstr ""

#: gpicsync-GUI.py:788
msgid "Adding "
msgstr ""

#: gpicsync-GUI.py:792
msgid "KMZ file created in pictures folder"
msgstr ""

#: gpicsync-GUI.py:794
msgid "Couldn't rename the zip file to kmz (maybe a previous file already exist)"
msgstr ""

#: gpicsync-GUI.py:796
msgid "To create a Google Earth kmz file you must either:"
msgstr ""

#: gpicsync-GUI.py:798
msgid "- select a folder you've already synchronized then select the KMZ Generator tool"
msgstr ""

#: gpicsync-GUI.py:806
msgid "Inspect a gpx file and show tracklog data."
msgstr ""

#: gpicsync-GUI.py:808
msgid "Select a gpx file"
msgstr ""

#: gpicsync-GUI.py:827
msgid "Select a gpx file first."
msgstr ""

#: gpicsync-GUI.py:830
msgid "Looking at "
msgstr ""

#: gpicsync-GUI.py:831
msgid "Number of valid track points found"
msgstr ""

#: gpicsync-GUI.py:834
msgid "Date"
msgstr ""

#: gpicsync-GUI.py:834
msgid "Time"
msgstr ""

#: gpicsync-GUI.py:835
msgid "Latitude"
msgstr ""

#: gpicsync-GUI.py:836
msgid "Longitude"
msgstr ""

#: gpicsync-GUI.py:837
msgid "Altitude"
msgstr ""

#: gpicsync.py:268
msgid "taken "
msgstr ""

#: gpicsync.py:269 gpicsync.py:289
msgid "writing best latitude/longitude match to picture: "
msgstr ""

#: gpicsync.py:270 gpicsync.py:276 gpicsync.py:290 gpicsync.py:297
msgid " time difference (s)= "
msgstr ""

#: gpicsync.py:275
msgid "no track point below the maximum time range "
msgstr ""

#: gpicsync.py:275 gpicsync.py:279 gpicsync.py:297
msgid " : WARNING: DIDN'T GEOCODE, "
msgstr ""

#: gpicsync.py:277
msgid " long="
msgstr ""

#: gpicsync.py:277
msgid "For information nearest trackpoint was at lat="
msgstr ""

#: gpicsync.py:279
msgid "no track point at this picture date "
msgstr ""

#: gpicsync.py:292
msgid "Warning: Picture date "
msgstr ""

#: gpicsync.py:293
msgid " and track point date "
msgstr ""

#: gpicsync.py:293
msgid " are different ! "
msgstr ""

#: gpicsync.py:297
msgid " no suitable track point below maximum time difference, "
msgstr ""

